app/bot.py

```diff
@@ -14,14 +14,11 @@
 MAP_API = os.getenv("MAP_API")
 
 
-logging.basicConfig(level=logging.INFO)
```

Comment:
Removing this logger will help future units save time because they wont have to shift through redundant code.